### ISSUE TITLE
Low Validation Accuracy with AVC-Test-Model

### ISSUE THREAD

**Issue by Simsso** [Simsso/NIPS-2018-Adversarial-Vision-Challenge#90]
While the ALP weights score ~72% on the Tiny ImageNet validation set, they achieve only ~56% against the challenge's `avc-test-model`:
```
Connecting to model sever... ✓
Server version 1.0.5
Checking model axis and bounds... ✓
Checking return value... ✓
Testing model accuracy on test samples...
100%|██████████████████████████████████████████████████████████████████████████| 500/500 [00:09<00:00, 52.94it/s]
The top-1 performance of your model is 56.4%.

All tests successful, have fun submitting!
```

The low accuracy has a significant impact on our reported robustness. This issue is about investigating the matter and achieving a higher accuracy here.
